pkg/utils/utils_test.go

```diff
@@ -840,7 +840,19 @@ var _ = Describe("Utils", Label("utils"), func() {
 				grub := utils.NewGrub(config)
 				err = grub.Install(target, rootDir, bootDir, constants.GrubConf, "", true, "")
 				Expect(err).To(HaveOccurred())
-				Expect(err.Error()).To(ContainSubstring("no such file"))
+				Expect(err.Error()).To(ContainSubstring("could not find any shim file to copy"))
+				// Create fake shim
+				err = fsutils.MkdirAll(fs, filepath.Join(rootDir, "/usr/share/efi/x86_64/"), constants.DirPerm)
+				Expect(err).ShouldNot(HaveOccurred())
+				err = fs.WriteFile(filepath.Join(rootDir, "/usr/share/efi/x86_64/", constants.SignedShim), []byte(""), constants.FilePerm)
+				Expect(err).ShouldNot(HaveOccurred())
+				err = fs.WriteFile(filepath.Join(rootDir, "/usr/share/efi/x86_64/shim.efi"), []byte(""), constants.FilePerm)
+				Expect(err).ShouldNot(HaveOccurred())
+				grub = utils.NewGrub(config)
+				err = grub.Install(target, rootDir, bootDir, constants.GrubConf, "", true, "")
+				Expect(err).To(HaveOccurred())
```

Comment:
[minor] maybe use `MatchError` or similar to make sure we don't match random errors but the expected one

Reply:
We match the error message below for this specific error that we want.